### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by vaishnavme**
##### Imp:  Sidebar Bug Fix
- Responsive design
- Code examples
- Complete site redesign

main branch preview: [main branch](https://icyui.netlify.app/)
dev. branch [Dev Branch](https://605afc79d37720d1b6a5a290--icyui.netlify.app/)
@im-rhlrvndrn 
@villdev
@amansethi00

Things need to get fixed
- Show real link for stylesheet
- Optimize images
- About me part, forgot to add this.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,138 +4,170 @@
...
-						<a href="#card">Card</a>
-					</li>
-					<li class="sidenav-items-link">
-						<a href="#navigation">Navigation</a>
-					</li>
-					<li class="sidenav-items-link">
-						<a href="#modal">Modal</a>
-					</li>
-				</ul>
+				<div id="sidebar-content" class="sidebar-content">
```
Comment (villdev) [inline index.html:45]: Should the class not be `sidebar__content` ? In line with `sidebar__header`.

File: index.html [hunk 2]
```diff
@@ -4,138 +4,170 @@
...
+						<div class="page__header d-flex flex-align-center">
+							<img src="images/logo.svg">
+							<h1>Build Ui Faster</h1>
+						</div>
+						<div class="page__info mb-3">
+							Set of simple, responsive CSS modules.
+						</div>
+						<div class="h4">Getting Started</div>
+						<p class="m-2">Simply add stylesheet link to your project header. <br/>Start Building !</p>
+						<input class="link__box" type="text" readonly value="&lt;link rel='stylesheet type='text/css' href='path_to/css'&gt;">
```
Comment (villdev) [inline index.html:107]: Does this link work as expected? Doesn't show correct one in sandbox

File: js/app.js [hunk 3]
```diff
@@ -1,13 +1,26 @@
...
-function toggleSideBar() {
-    sidebar.classList.toggle("active")
-    customMenu.classList.toggle("right-move")
-}
+// Active section link
+let mainNavLinks = document.querySelectorAll("#sidebar-content li a");
+let mainSection = document.querySelectorAll("main section")
 
-toggleButton.addEventListener("click", toggleSideBar)
+window.addEventListener("scroll", event => {
```
Comment (villdev) [inline js/app.js:14]: Doesn't work for the modal link in sidebar.

File: index.html (hunk 1 above)
Comment (villdev) [inline index.html]: Should the class not be `sidebar__content` ? In line with `sidebar__header`.

File: index.html (hunk 2 above)
Comment (villdev) [inline index.html]: Does this link work as expected? Doesn't show correct one in sandbox

File: js/app.js (hunk 3 above)
Comment (villdev) [inline js/app.js:14]: Doesn't work for the modal link(last one) in sidebar.

File: style/main.css [hunk 4]
```diff
@@ -0,0 +1,460 @@
...
+	visibility: hidden;
+	transition: opacity .4s, max-height .4s, visibility .4s;
+}
+.sidebar-content.active {
+	max-height: 90vh;
+	opacity: 1;
+	overflow-y: auto;
+	visibility: visible;
+}
+.sidebar-content li a.current {
```
Comment (villdev) [inline style/main.css]: Maybe make the active class(`.current`) more emphasized imo. Upto you though.

**Review by villdev — COMMENTED** [review]
Great work! That's all I could find at a glance.

File: style/main.css [hunk 5]
```diff
@@ -0,0 +1,460 @@
...
+		width: 50vw;
+		background-color: #ffffff;
+		z-index: -1;
+		border-radius: 0.5rem;
+	}
+}
+/*=======================================
+=================Colors==================
+=========================================*/
+.color__block {
```
Comment (amansethi00) [inline style/main.css]:
```suggestion
.color__block {
Class name do not go handy with the properties inside,name maybe w80-h50-card,feel free to choose any other name 
```

File: style/main.css [hunk 6]
```diff
@@ -0,0 +1,460 @@
...
+	height: 1rem;
+	bottom: 0.8rem;
+	right: 0.1rem;
+	border-radius: 50%;
+	border: 3px solid var(--white);
+}
+.status-active {
+	background-color: var(--success);
+}
+.status-idl {
```
Comment (amansethi00) [inline style/main.css]: go with status-warning name instead status-id1,as it will look consistent

**Review by amansethi00 — COMMENTED** [review]
Great work Vaishnav ,class naming is consistent throughout the code

File: index.html [hunk 7]
```diff
@@ -4,138 +4,178 @@
...
 	</head>
 	<body>
-		<section class="wrapper d-flex">
-			<nav id="sidebar">
-				<div class="custom-menu pt-3">
-					<button type="button" id="sidebarCollapse">
-						<span class="bars"></span>
-						<span class="bars"></span>
-						<span class="bars"></span>
+		<section id="container" class="container">
```
Comment (imrhlrvndrn) [inline index.html:26]: change thsi tag

File: style/main.css [hunk 8]
```diff
@@ -0,0 +1,467 @@
...
+	align-items: center;
+	justify-content: center;
+	position: absolute;
+	top: 2px;
+	right: 5px;
+	border-radius: 50%;
+	width: 1.2rem;
+	height: 1.2rem;
+}
+.label-bell {
```
Comment (imrhlrvndrn) [inline style/main.css]: Update the names

**Review by imrhlrvndrn — CHANGES_REQUESTED** [review]

**Pull request merged**